common/tests/org.jboss.tools.common.el.core.test/src/org/jboss/tools/common/el/core/test/resolver/TypeInfoCollectorTest.java

```diff
@@ -132,6 +132,19 @@ public void testProperties() throws CoreException {
 		assertTrue(ns2.contains("getState")); //$NON-NLS-1$
 	}
 
+	public void testPropertiesInEnum() throws CoreException {
+		IJavaProject jp = JavaCore.create(project2);
+		String className = "test.TestF"; //$NON-NLS-1$
+		IType bean = jp.findType(className);
+		TypeInfoCollector.TypeInfo typeInfo = new TypeInfoCollector.TypeInfo(bean, null, false);
+		TypeInfoCollector collector = typeInfo.getTypeCollector(false, false);
+		MemberInfo status = getMethod(collector, "getStatus");
+		assertNotNull(status);
+		collector = status.getTypeCollector(false, false);
+		MemberInfo name = getMethod(collector, "name");
+		assertNotNull(name);
+	}
+
```

Comment:
Ah, sorry, seems I've copy/pasted the wrong message in the issue. I've updated the issue now.
The point of the 2nd example was to show that even a standard property (label) isn't detected in some cases.